RZBRP100/iocBoot/iocRZBRP100-IOC-01/config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
+<ioc_desc>Razorbill RP100 Strain Cell PSU</ioc_desc>
+<ioc_details></ioc_details>
+<macros>
+<macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" />
```

Comment:
Wiki says this is a USB device but this looks like standard serial port setup - which is it?

Reply:
Both.  The USB driver creates a virtual COM port on the host, enabling it to be accessed as a standard serial port.  More details have been added to the [Wiki page](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Razorbill-RP100-Strain-Cell-PSU).